docs/sdk_python/delete_file.mdx

```diff
@@ -5,10 +5,13 @@ sidebarTitle: Remove a File
 
 ## Description
 
-In MindsDB, files are treated as tables. These are stored in the default `files` database. To delete a file, you must save this `files` database into a variable and then, run the `delete_table()` function on it.
+In MindsDB, files are treated as tables. These are stored in the default `files` database. To delete a file, you must save this `files` database into a variable and then, run the `drop_table()` function on it.
```

Comment:
drop is a new feature and I didn't implement function for old syntax.
It means only this option is available:
`files_db.tables.drop(name)`
And no `drop_table` function exists